svo (custom prompt, serverside, peopletracker).xml

```diff
@@ -317,7 +317,7 @@ cpp.compute_gametargethp = function()
 end
 
 cpp.compute_weaponmastery = function()
-  return stats.weaponmastery or 0
+  return classinfo.weaponmastery or 0
 end
 
 cpp.compute_power = function()
```

Comment:
Line 324 should be changed from 'stats.shin or stats.kai' to 'classinfo.shin or classinfo.kai'